hnn_core/feed.py

```diff
@@ -173,12 +190,15 @@ def __create_extgauss(self):
         val_gauss.sort()
         # if len(val_gauss)>0: print('val_gauss:',val_gauss)
         # Convert array into nrn vector
-        self.eventvec.from_python(val_gauss)
-        return self.eventvec.size() > 0
+        self.nrn_EventVec.from_python(val_gauss)
+        return self.nrn_EventVec.size() > 0
 
-    def __create_extinput(self):
-        """Creates the ongoing external inputs (rhythmic)."""
-        # print("__create_extinput")
+    def __create_common_input(self):
```

Comment:
I did this already but not completely. We should avoid methods to have double underscores. It mangles the function names. Just single underscore is good for a private method